packages/react-examples/src/react/ContextualMenu/ContextualMenu.Checkmarks.Example.tsx

```diff
@@ -104,6 +104,7 @@ export const ContextualMenuCheckmarksExample: React.FunctionComponent = () => {
           },
           text: 'Split Button',
           canCheck: true,
+          checked: true,
```

Comment:
added checked true here as it being disabled, there is no way to see the checkmark color